Task_DEV_4/Task_DEV_4/EntryPoint.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+
+namespace Task_DEV_4
+{
+    class Program
```

Comment:
If your file is named like EntryPoint it would be better to rename `class Program` to` class EntryPoint`

Reply:
Done.